### ISSUE TITLE
Blank UI on empty notes

### ISSUE THREAD

**Issue by imujjwal96** [BrimeNotes/android#93]
### This is a
- [ ] Bug report
- [x] Feature request

### Current behaviour

When the section Public Notes, Private Notes and Trash is empty, the UI is blank.

| Public Notes | Trash |
|-----------------|---------|
| ![screenshot_20170721-201812](https://user-images.githubusercontent.com/8245662/28470959-1207a19e-6e59-11e7-9cef-58e45a7f7f2c.png) | ![screenshot_20170721-210552](https://user-images.githubusercontent.com/8245662/28470958-1206f3b6-6e59-11e7-8f70-2bd30ce99a46.png)

### Expected behaviour

Handle this beautifully by showing some messages/images

**Comment by imujjwal96** [comment]
From what i know, didn't we already show something when these were blank @syedsaadh ?

**Comment by syedsaadh** [comment]
Yes we do have have a resource for this in drawable named `empty_buddy`. We can use it instead of this blank screen

**Comment by nervehammer** [comment]
@syedsaadh why we want to have _empty_buddy_ in trash fragment :D
